configure.ac

```diff
@@ -67,6 +67,16 @@ case $host in
   ;;
 esac
 
+AC_ARG_ENABLE([syscall-sandbox],
+  [AS_HELP_STRING([--enable-syscall-sandbox],
```

Comment:
Why not make syscomp opt-out? I think secure by default is a better idea :-)

Reply:
Maybe it would make sense to default it to on in linux in master and if there is concern that it's not mature yet, switch it to warning only or disable it before a release.  This is the sort of thing that should be default-on unless it turns out to have bad performance consequences (and maybe even then, depending on how bad they are).

Reply:
IIRC there was a discussion about something similar here #17419
However it is a bit weird and in transparent IMO to have things enabled in a dev build and others in a release build (beside some warnings of course).

Reply:
I didn't really intended to go as far as suggesting some differences between "dev build" and "release build" on an ongoing basis so much as just being eager to turn it off if there are issues.  Otherwise you get a compatibility chicken and egg:  You don't know if it will cause problems for some users without deploying it, but you don't want to deploy it unless you know it won't cause problems.

Reply:
I don't think enabling it by default should be considered in the PR first introducing this. It is absolutely something that needs experimentation and testing and fine-tuning, in different environments.